packages/server/src/games.ts

```diff
@@ -199,6 +200,11 @@ export async function handleMoveAndTime(
 
   emitGame(game.id, game.players?.length ?? 0, game_obj, timeControl);
 
+  if (game_obj.phase == "gameover" && game.variant == "quantum") {
+    // user ranking value is only for quantum right now
+    await updateRatings(game, game_obj);
```

Comment:
I'm actually not sure if we should `await` this one.  My thought process is this:

- There is no return value for us to wait for
- If there is a (rare) error in `updateRatings`, it's not such a big deal.  We have already updated the game object to reflect the move.

Reply:
Can't say I've seen many unawaited async function calls that were not mistakes. But I think it's probably not a huge deal either way, since the end of a game happens only so often.

Reply:
That's a fair point, and I realize now the linter may complain if we don't await ([no-floating-promises](https://typescript-eslint.io/rules/no-floating-promises/)). Let's leave as-is!